src/compas_viewer/scene/pointobject.py

```diff
@@ -8,12 +8,21 @@
 class PointObject(ViewerSceneObject, GeometryObject):
     """Viewer scene object for displaying COMPAS Point geometry.
 
+    Parameters
+    ----------
+    point : :class:`compas.geometry.Point`
+        The point geometry to display.
+    show_points : bool, optional
+        Whether to display the point in the viewer. Default is True.
+
     See Also
     --------
     :class:`compas.geometry.Point`
     """
 
     def __init__(self, point: Point, **kwargs):
+        if kwargs["show_points"] is None:
+            kwargs["show_points"] = True
```

Comment:
please stop using `kwargs` for everything.
if you know the parameter, add it by name and document it...